kinase_focused_fragment_library/fragmentation/classes.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, mol=None, atomNumbers=None, subpocket=None, structure=None, c
         self.center = center
         self.subpocket = subpocket
         self.structure = structure
-        self.environment = environment
+        self.environment = environment or []
```

Comment:
I do not fully understand this. If we pass `environment=None`, we want to set `self.environment=[]`?

Shouldn't the environment always be a list now (if I understand the PR comments correctly)? If so, to me this would make more sense:

Update docstring
```
    Attributes
    ----------
   ...
    environment: list[str]
        List of BRICS environment types of the fragment (based on RDKit definitions)

```

And then in `__init__`:
```
if environment is None:
    self.environment = []
elif isinstance(environment, str):
    self.environment = [environment]
elif isinstance(environment, list):
    self.environment = environment
else:
    raise ValueError(f"Input type {} of `environment` can only be str or list[str]")
```

Reply:
Thanks a lot, I've updated the doc string and changed the init method :)